platform/viewer/src/lib/DICOMSR/srUtils.js

```diff
@@ -0,0 +1,71 @@
+import OHIF from '@ohif/core';
+
+const { studyMetadataManager } = OHIF.utils;
```

Comment:
`studyMetadataManager` isn't really a "utility". It's a bit odd that we expose it this way. If any of these methods really need the whole metadataManager, can we pass it in as a parameter? If they don't, let's scope what we pass in to only the information each function needs.